src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/adjourncase/AdjournCaseMidEventDueDateService.java

```diff
@@ -0,0 +1,31 @@
+package uk.gov.hmcts.reform.sscs.ccd.presubmit.adjourncase;
+
+import static java.util.Objects.nonNull;
+import static uk.gov.hmcts.reform.sscs.util.DateTimeUtils.isDateInTheFuture;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.sscs.ccd.callback.Callback;
+import uk.gov.hmcts.reform.sscs.ccd.callback.PreSubmitCallbackResponse;
+import uk.gov.hmcts.reform.sscs.ccd.domain.SscsCaseData;
+
+@Component
+@Slf4j
+public class AdjournCaseMidEventDueDateService {
+
+    public PreSubmitCallbackResponse<SscsCaseData> validateAdjournCaseDirectionsDueDateIsInFuture(Callback<SscsCaseData> callback) {
+
+        SscsCaseData sscsCaseData = callback.getCaseDetails().getCaseData();
+
+        PreSubmitCallbackResponse<SscsCaseData> preSubmitCallbackResponse = new PreSubmitCallbackResponse<>(sscsCaseData);
```

Comment:
if we're creating this as a validation service, I don't think it needs to know about the callback response
its input should be caseData, and its output a boolean